person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = id
```

Comment:
- [ ] Kindly give id value like `@id = Random.rand(1..1000)` as per the requirements.

Reply:
@sja-thedude thanks for the feedback.